### ISSUE TITLE
NSI querySummary request fails silently

### ISSUE THREAD

**Issue by jmacauley** [esnet/oscars#197]
When an NSI querySummary request is issued to the Netlab OSCARS for a non-existing reservation the querySummaryConfirmed is correctly returned, however, when a querySummary request targeting an existing reservation is issued, neither a querySummaryConfirmed nor error are returned.

**Comment by jmacauley** [comment]
querySummarySync behaves similarly but returns a soap:Fault when an existing connectionId is specified:

```
11:07:32,648
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns="">
    <soap:Body>
        <soap:Fault>
            <faultcode>soap:Server</faultcode>
            <faultstring>Fault occurred while processing.</faultstring>
        </soap:Fault>
    </soap:Body>
</soap:Envelope>
```

**Comment by haniotak** [comment]
caused by NPE; should be fixed in 1.0.10
